spec.html

```diff
@@ -34663,7 +34663,7 @@ <h1>Static Semantics: Early Errors</h1>
         <emu-grammar>AtomEscape :: DecimalEscape</emu-grammar>
         <ul>
           <li>
-            It is a Syntax Error if the CapturingGroupNumber of |DecimalEscape| is larger than _NcapturingParens_ (<emu-xref href="#sec-notation"></emu-xref>).
+            It is a Syntax Error if the CapturingGroupNumber of |DecimalEscape| is larger than LeftCapturingParensWithin(the |Pattern| containing |AtomEscape|).
```

Comment:
I agree with @jmdyck that it feels odd to not have "number of" or "count" in the SDO name somehow. Maybe LeftCapturingParensWithinCount?

Reply:
`CountLeftCapturingParensWithin` is mutually satisfactory per editor call.

Reply:
Done (for both AOs).